sharktank/sharktank/layers/paged_llama_attention_block.py

```diff
@@ -117,6 +117,16 @@ def __init__(
                     theta.optional_tensor(f"{attn_name}.q_output"),
                 )
 
+        # Attention sink handling: Load from theta if available, otherwise set to None
+        if "attn_sinks" in theta.keys:
+            sink_tensor = (
+                theta("attn_sinks").as_torch().to(dtype=config.attention_dtype)
```

Comment:
How come the `.to(dtype=config.attention_dtype)` is needed? Why isn't it stored as the correct dtype in the theta?